lib/components/proposals/ProposalsUI.jsx

```diff
@@ -1,103 +1,182 @@
-            <h6 className='mb-2'>{t('joinTheDiscussion')}</h6>
-            <p>
-              <Trans
-                i18nKey='checkoutTheForumAndDiscord'
-                defaults='Check out our <linkToForum>forum</linkToForum> and our <linkToDiscord>Discord</linkToDiscord> to stay up to date.'
-                components={{
-                  linkToForum: <ExternalLink underline href={POOLTOGETHER_GOV_FORUM_URL} />,
-                  linkToDiscord: <ExternalLink underline href={DISCORD_INVITE_URL} />
-                }}
-              />
-            </p>
-          </Card>
-          <Card className='w-full sm:w-1/2 sm:ml-4'>
-            <h6 className='mb-2'>{t('wantToVoteGasFree')}</h6>
-            <p>
-              <Trans
-                i18nKey='depositIntoPoolPoolToVoteGasFree'
-                defaults='Deposit into the <poolPoolLink>POOL Pool</poolPoolLink> to vote without transaction fees on <snapshotLink>Snapshot</snapshotLink>.'
-                components={{
-                  poolPoolLink: <ExternalLink underline href={POOLPOOL_URL} />,
-                  snapshotLink: <ExternalLink underline href={POOLPOOL_SNAPSHOT_URL} />
-                }}
-              />
-            </p>
-          </Card>
-        </div>
-      </div>
+      <Tabs className='justify-between sm:justify-start sticky bg-body top-20 pt-8 mb-8 pb-4'>
+        {TABS.map((tab) => (
+          <tab.tabView key={tab.id} tab={tab} currentTab={currentTab} setTab={setTab} />
+        ))}
+      </Tabs>
 
-      <ProposalsList />
-
-      <AddGovernanceTokenToMetaMask />
+      {TABS.map((tab) => (
+        <ContentPane key={tab.id} isSelected={tab.id === currentTab}>
+          <tab.view tab={tab} />
+        </ContentPane>
+      ))}
     </>
   )
 }
+
+const CommonTabView = (props) => {
+  const { tab } = props
+
+  const { isFetched, sortedProposals } = useAllProposalsSorted()
+
+  let count = 0
+  if (isFetched) {
+    const proposals = []
+    tab.proposalStates.forEach((proposalState) => {
+      proposals.push(...sortedProposals[proposalState])
+    })
+    count = proposals.length
+  }
+
+  return <TabView {...props} count={count} />
+}
+
+const SnapshotTabView = (props) => {
+  const { data: proposals, isFetched, error } = useSnapshotProposals()
+
+  let count = 0
+  if (isFetched && !error) {
+    count = proposals.length
+  }
+
+  return <TabView {...props} count={count} />
+}
+
+const TabView = (props) => {
+  const { tab, count, currentTab, setTab } = props
+  const isSelected = tab.id === currentTab
+
+  return (
+    <div className='flex sm:mr-6 lg:mr-8 last:mr-0'>
+      <Tab
+        key={tab.id}
+        isSelected={isSelected}
+        className='flex mx-2'
+        onClick={() => setTab(tab.id)}
+      >
+        <div className='flex'>
+          {tab.title}
+          {count > 0 && (
+            <CountBadge
+              count={count}
+              bgClassName='bg-highlight-1'
+              className={classnames('ml-2 my-auto', {
```

Comment:
We should update `CountBadge` to use the `tracking-tight` tailwind class, will make this:
<img width="31" alt="Screenshot 2021-06-21 at 13 09 40" src="https://user-images.githubusercontent.com/35717/122821670-fbb58400-d291-11eb-9743-1f2314976b9b.png">

look like this:
<img width="21" alt="Screenshot 2021-06-21 at 13 09 29" src="https://user-images.githubusercontent.com/35717/122821687-ff490b00-d291-11eb-92f3-454b681ff2b9.png">

Also there is an newly updated CountBadge in the Flagship that I re-did about two weeks ago. We may want to consider moving that code from there over to the components lib.